src/core/Akka.Cluster.Tests/ClusterSpec.cs

```diff
@@ -192,6 +194,76 @@ public void A_cluster_must_complete_LeaveAsync_task_upon_being_removed()
             _cluster.LeaveAsync().IsCompleted.Should().BeTrue();
         }
 
+        [Fact(Skip = "This behavior not yet supported.")]
```

Comment:
Might need to compare the output of `FillLocal(address)` to `SelfAddress`, although I'd need to look at the `FillLocal` function again to say for that sure :p